### PR TITLE
chore: remove outdated descriptions from README.mds

### PR DESCRIPTION
**Pull request by polaroi8d**

### CODE REVIEW

File: agent/README.md [hunk 1]
```diff
@@ -1,38 +1,3 @@
...
-| NODE_ID                 | ID of the node                                                                                            | cb7e9573-9a43-4d5b-8005-eb8bb7a423c4"` |
-| READ_HEADER_TIMEOUT     | Gin read header timeout, configured to protect against Slowloris type attacks (form of denial of service) | 15s                                    |
-| REGISTRY                | Default registry address                                                                                  | index.docker.io                        |
-| REGISTRY_PASSWORD       | Password of the remote registry where DAgent is located                                                   | *none*                                 |
-| REGISTRY_USERNAME       | User of the remote registry where DAgent is located                                                       | *none*                                 |
+# dyrector.io platform: agent
 
+We are working on the source code documentation until, please use the root [README.md](../../README.md) for further information or check our official [documentation](https://docs.dyrector.io/) site.
```
Comment (robot9706) [inline agent/README.md:3]: `We are working on the source code documentation, until then please use the root ...` sounds a bit better in my opinion 😇

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: agent/README.md (hunk 1 above)
Comment (peterambrus3) [inline agent/README.md:3]: I think one ../ is enough to reach the root readme, or not?

Comment (polaroi8d) [reply agent/README.md:3]: Fixed.

**Review by peterambrus3 — APPROVED** [review]
lgtm

File: protobuf/README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
-# Protorepo
+# dyrector.io platform: protorepo
 
-Containing all dyrector.io related proto files
-and generating proto packages
+We are working on the source code documentation until, please use the root [README.md](../../README.md) for further information or check our official [documentation](https://docs.dyrector.io/) site.
```
Comment (robot9706) [inline protobuf/README.md:3]: `../` here as well.

**Review by robot9706 — CHANGES_REQUESTED** [review]

**Review by robot9706 — APPROVED** [review]
LGTM 😎😎

**Pull request merged**